app/pages/SmartDisplay.tsx

```diff
@@ -0,0 +1,19 @@
+import { useLoaderData } from "@remix-run/react";
+
+function SmartDisplay() {
+
+    const data = useLoaderData()
+  
+    console.log(data);
+  
+    return (
+      <div>
+        <h1>Le stop ID est : </h1>
+        <h2>{data.params.stopIds}</h2>
```

Comment:
Could you deconstruct the data obj to access `stopIds` ? 

Also using a ternary operator (https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#:~:text=L'op%C3%A9rateur%20(ternaire)%20conditionnel,%2Fif...else%20.) can you try to display : 
if (stopIds) ==> (
```
<h1>Le stop ID est : </h1>
<h2>{data.params.stopIds}</h2>
```
)
if (!stopIds) ==> (
`<h1>Error Message</h1>`
)

Reply:
I can try